spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/PublisherCallbackChannel.java

```diff
@@ -34,26 +34,19 @@
 	String RETURN_CORRELATION_KEY = "spring_listener_return_correlation";
 
 	/**
-	 * Adds a {@link Listener} and returns a reference to
-	 * the pending confirms map for that listener's pending
-	 * confirms, allowing the Listener to
-	 * assess unconfirmed sends at any point in time.
-	 * The client must <b>NOT</b> modify the contents of
-	 * this array, and must synchronize on it when
-	 * iterating over its collections.
-	 *
+	 * Adds a {@link Listener}
 	 * @param listener The Listener.
-	 * @return A reference to pending confirms for the listener
 	 */
-	SortedMap<Long, PendingConfirm> addListener(Listener listener);
+	void addListener(Listener listener);
```

Comment:
M-m-m. Seems for me Andy Wilkinson won't be happy to have such a braking change in the minor release...
Although we may hope that no one has tests for this `PublisherCallbackChannel` :smile: